tests/pkg/fixture/fixture.go

```diff
@@ -127,6 +127,77 @@ func GetTidbCluster(ns, name, version string) *v1alpha1.TidbCluster {
 	}
 }
 
+func GetFullTidbCluster(ns, name, version string) *v1alpha1.TidbCluster {
```

Comment:
reanme `GetFullTidbCluster`  to `GetTidbClusterWithTiFlash`

`FullTidbCluster` is not correct, because `TidbCluster` includes Pump, CDC (later).

Reply:
Updated.